frontend/packages/console-app/src/components/modals/clone/clone-pvc-modal.tsx

```diff
@@ -119,23 +121,29 @@ const ClonePVCModal = withHandlePromise((props: ClonePVCModalProps) => {
   return (
     <Form onSubmit={submit}>
       <div className="modal-content modal-content--no-inner-scroll">
-        <ModalTitle>Clone</ModalTitle>
+        <ModalTitle>{t('console-app-package~Clone')}</ModalTitle>
```

Comment:
I don't mind having one namespace per package, but I'd just make it `console-app` here. It doesn't need to follow a specific convention since it's not a dynamic plugin.

Reply:
All set @spadgett!